module/VuFind/src/VuFind/Cover/Router.php

```diff
@@ -106,6 +106,12 @@ public function getUrl(RecordDriver $driver, $size = 'small',
             : ['size' => $size];
         $handlers = $this->coverLoader->getHandlers();
         $ids = $this->coverLoader->getIdentifiersForSettings($settings);
+        if ($testLoadImage) {
```

Comment:
How should this feature interact with the "direct URL" mechanism? It seems to me that for providers that allow direct URLs, we can't really test them from here.. and the code currently seems to favor direct URLs over dynamic URLs when available. Thus, I'm wondering if this check should come AFTER the handlers loop that searches for direct URLs, and should only execute if no direct URLs are found. Otherwise, I think we may end up doing some expensive dynamic URL checking that won't mean anything because the direct URL gets used instead.